cfgov/paying_for_college/models/search.py

```diff
@@ -0,0 +1,32 @@
+from paying_for_college.documents import SchoolDocument
+from search.elasticsearch_helpers import make_safe
+
+
+class SchoolSearch:
+    def __init__(self, search_term):
+        self.search_term = make_safe(search_term).strip()
+        self.results = []
+
+    def autocomplete(self):
+        if self.search_term != '':
+            search = SchoolDocument.search().query(
+                'match', autocomplete=self.search_term)
+            total_results = search.count()
+            self.results = [
+                {'question': str(result.autocomplete), 'url': result.url}
+                for result in search[:total_results]
```

Comment:
This appears to be copied from ask-cfpb. The return values need to match what the page is expecting:
id, schoolname, city, state.